### ISSUE TITLE
[Help]: How to check if unwrapping is successful or is running

### ISSUE THREAD

**Issue by swjiaotong-jyliu** [gmtsar/gmtsar#160]
Dear Sir or Madam
I began 9 examples about Hawaii  with different  **threshold_snaphu** and **filter_wavelength** .
4 examples maybe finish，but 5 examples didn't find the unwrapp.grd.
![IR7U_SSN5O8%48 KD9QF139](https://user-images.githubusercontent.com/86451834/123364511-1a27c200-d5a7-11eb-80da-aa4f06b03e8f.png)

OS Linux version 3.10.0-957.27.2.el7.x86_64
GMTSAR version 6.0

merge folder in the completed example
![c](https://user-images.githubusercontent.com/86451834/123369476-64fa0780-d5b0-11eb-9f15-d5dfa7255441.png)

merge folder in the uncompleted example
![un](https://user-images.githubusercontent.com/86451834/123369475-64fa0780-d5b0-11eb-9863-3c76176d878c.png)

I didn't find unwrapp.grd.
Maybe I should reset the parameters?

Thanks for your answer.

**Comment by Xiaohua-Eric-Xu** [comment]
Is there any log file you could provide when you run sanphu.csh?

**Comment by swjiaotong-jyliu** [comment]
First of all, thank you very much for your reply.
The log.txt looks right.
In fact，one of them was completed  9 hours age，which **threshold_snaphu** = 0.3  and **filter_wavelength** = 0.5.

The log.txt as follow

nohup:
vv
1
Linking files for Subswath 1 ...
Linking files for Subswath 2 ...
Linking files for Subswath 3 ...
[1] 140114
[2] 140115
[3] 140116
[3]  - Done                          p2p_processing.csh S1_TOPS s1a-iw3-slc-vv-20180502t043028-20180502t043053-021721-025793-006 s1a-iw3-slc-vv-20180514t043029-20180514t043054-021896-025d31-006 config.tops.txt >& log
[2]  + Done                          p2p_processing.csh S1_TOPS s1a-iw2-slc-vv-20180502t043027-20180502t043052-021721-025793-005 s1a-iw2-slc-vv-20180514t043028-20180514t043053-021896-025d31-005 config.tops.txt >& log
[1]  + Done                          p2p_processing.csh S1_TOPS s1a-iw1-slc-vv-20180502t043026-20180502t043054-021721-025793-004 s1a-iw1-slc-vv-20180514t043027-20180514t043055-021896-025d31-004 config.tops.txt >& log

Merging START
Number of Files to be merged is 3 
Reading in the grids...
Writing the grid files..Size(8456x6761)...
Number of Files to be merged is 3 
Reading in the grids...
Writing the grid files..Size(8456x6761)...
Number of Files to be merged is 3 
Reading in the grids...
Writing the grid files..Size(8456x6761)...
Merging END

Recomputing the projection LUT...
SNAPHU.CSH - START
threshold_snaphu: 0.3
Working on line  1 ... ..............  ..6722（I omitted the number in the middle）

Setting search radius to be 300 ...
Reading in original grid...
Interpolating to nearest neighbour...
  -1620568906 number of searches used ...
WRITING GRID IMAGE: Width x Heihgt = 8456 x 6761...
unwrapping phase with snaphu - higher threshold for faster unwrapping 

snaphu v2.0.1
32 parameters input from file /opt/GMTSAR/share/gmtsar/snaphu/config/snaphu.conf.brief (297 lines total)
Reading wrapped phase from file phase.in
No weight file specified.  Assuming uniform weights
Reading correlation data from file corr.in
Calculating smooth-solution cost parameters
Building range cost arrays
Building azimuth cost arrays
Initializing flows with MST algorithm
Running approximate minimum spanning tree solver
Running nonlinear network flow optimizer
Maximum flow on network: 21
Number of nodes in network: 57155801
Flow increment: 1  (Total improvements: 0)
446 incremental costs clipped to avoid overflow (0.000%)
Found 1 valid set(s) of connected pixels
Source 0 row, col = -2, -2


Treesize: 1          Pivots: 0           Improvements: 0   
......
Treesize: 53406801   Pivots: 263958787   Improvements: 103550



I decided to do cutting rigion first. 
I am sorry to bother you.
Thank you very much for your reply to my impulsed question.
